tests/integ/test_local_mode.py

```diff
@@ -366,3 +366,32 @@ def test_mxnet_local_data_local_script():
         mx.delete_endpoint()
         time.sleep(5)
         fcntl.lockf(local_mode_lock, fcntl.LOCK_UN)
+
+
+@pytest.mark.continuous_testing
+def test_local_transform_mxnet(sagemaker_local_session, tmpdir):
```

Comment:
nitpick: s/transform/batch_transform (just in case anyone thinks of transform as real-time inference since the input-predict-output cycle is "transform")